cn/docs/concepts/cluster-administration/device-plugins.md

```diff
@@ -0,0 +1,96 @@
+---
+approvers:
+title: 设备插件
+描述： 使用 Kubernetes 设备插件框架来为 GPUs、 NICs、 FPGAs、 InfiniBand 和类似的需要供应商特定设置的资源实现插件。
+---
+
+{% include feature-state-alpha.md %}
+
+{% capture overview %}
+从1.8版本开始，Kubernetes 提供了一套
+[设备插件框架](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/resource-management/device-plugin.md)，
+使得供应商能够在不改动 Kubernetes 核心代码的情况下，向 kubelet 发布它们的资源。
+供应商可以实现一个手动或以 DaemonSet 形式部署的插件，而不是编写自定义的 Kubernetes 代码。
+插件的目标设备包括 GPUs、 High-performance NICs、 FPGAs、 InfiniBand 
+和其他类似的可能需要供应商特定的初始化和设置的计算资源。
+{% endcapture %}
+
+{% capture body %}
+
+## 设备插件注册
+
+设备插件功能通过 `DevicePlugins` 功能入口控制， 该功能默认是禁用的。
+当设备插件功能被启用时，kubelet 会对外提供一个 `Registration` gRPC 服务：
+
+```gRPC
+service Registration {
+	rpc Register(RegisterRequest) returns (Empty) {}
+}
+```
+设备插件通过该 gRPC 服务将自身注册到 kubelet 。
+注册过程中，设备插件需要发送:
+
+  * 它的 Unix 套接字名称。
+  * 所基于的设备插件 API 版本。
+  * 希望发布的 `ResourceName` 。 这里的 `ResourceName` 需要符合
+    [扩展资源命名方案](https://github.com/kubernetes/kubernetes/pull/48922)，
+    形如 `vendor-domain/resource` 。
+    例如，Nvidia GPU 资源被发布为 `nvidia.com/gpu` 。
+
+注册成功后，设备插件将其管理的设备列表发送至 kubelet ，然后 kubelet 负责将这些资源作为 kubelet 节点状态更新的一部分，通知 apiserver 。
+例如， 设备插件注册 `vendor-domain/foo` 到 kubelet ，
+并上报了节点上的两个健康的设备后，节点状态将更新， 发布2个 `vendor-domain/foo` 。
+
+然后，开发者可以在 [容器](/docs/api-reference/{{page.version}}/#container-v1-core)
+规格中通过使用与
+[模糊整数型资源](/docs/tasks/configure-pod-container/opaque-integer-resource/)
+中同样的流程来请求使用设备。
+在1.8版本中， 扩展资源仅支持整型的资源，且容器规格中声明的 `limit` 与 `request` 必须相等。
+
+## 设备插件实现
+
+设备插件的工作流程一般包括以下步骤:
+
+* 初始化。 在这个阶段，设备插件执行供应商特定的初始化和设置，以确保设备处于就绪状态。
```

Comment:
The first step in initialization is registration.

Reply:
Nope, a plugin register itself to kubelet after its initialization and service startup